config-sources/zookeeper/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.smallrye</groupId>
+    <artifactId>zookeeper-config-source</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>SmallRye: ZooKeeper ConfigSource</name>
+
+    <properties>
+        <version.microprofile.config>1.1</version.microprofile.config>
+        <version.curator>2.12.0</version.curator>
+        <version.junit>4.12</version.junit>
+        <version.assertj>3.10.0</version.assertj>
+        <version.surefire>2.21.0</version.surefire>
+        <version.arquillian>1.1.10.Final</version.arquillian>
+        <version.wildfly.swarm>2018.5.0</version.wildfly.swarm>
+        <version.shrinkwrap>2.2.0</version.shrinkwrap>
+
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <failOnMissingWebXml>false</failOnMissingWebXml>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>${version.surefire}</version>
+                <configuration>
+                    <test>TestSuite</test>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.apache.curator</groupId>
+            <artifactId>curator-recipes</artifactId>
+            <version>${version.curator}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.eclipse.microprofile.config</groupId>
+            <artifactId>microprofile-config-api</artifactId>
+            <version>${version.microprofile.config}</version>
+        </dependency>
+
+        <dependency>
```

Comment:
This needs to be pulling in smallrye-config, and nothing from WF Swarm/Thorntail